bridge/client/app/_components/ktb-create-service/ktb-create-service.component.spec.ts

```diff
@@ -7,7 +7,7 @@ import { AppModule } from '../../app.module';
 import { NotificationsService } from '../../_services/notifications.service';
 import { NotificationType } from '../../_models/notification';
 import { HttpErrorResponse } from '@angular/common/http';
-import { ApiService } from '../../_services/api.service';
+import { APIService } from '../../_services/api.service';
```

Comment:
I think, this is an exported class name, why do we need this change? It's unrelated to `golint` warnings.